apps/src/templates/studioHomepages/ContentBox.jsx

```diff
@@ -48,7 +49,7 @@ const styles = {
   }
 };
 
-const CollapsibleSection = React.createClass({
+const ContentBox= React.createClass({
```

Comment:
This name is now fairly generic. It's likely worth having a comment describing what makes something a ContentBox (i.e. what this component is and when it should/should not be used).